app/controllers/public_token_magic_links_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PublicTokenMagicLinksController < ApplicationController
+  def show
```

Comment:
http://localhost:3000//tokens/show_magic_link

=> undefined method `+' for nil:NilClass